test/test_util.py

```diff
@@ -385,4 +385,24 @@ def test_restart_unexpected_dead_sql_thread() -> None:
 
 def test_trabackup_version() -> None:
```

Comment:
Would be nice to fix this typo :)
```suggestion
def test_xtrabackup_version() -> None:
```